tests/modules/nf-core/semibin/singleeasybin/main.nf

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BOWTIE2_BUILD } from '../../../../../modules/nf-core/bowtie2/build/main.nf'
+include { BOWTIE2_ALIGN } from '../../../../../modules/nf-core/bowtie2/align/main.nf'
+include { SEMIBIN_SINGLEEASYBIN } from '../../../../../modules/nf-core/semibin/singleeasybin/main.nf'
+
+workflow test_semibin_singleeasybin {
+    input = [
+        [ id:'test', single_end:true ], // meta map
+        [
+            file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true)
+        ]
+    ]
+    
+    fasta = [
+        [ id:'test'],
+        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+    ]
+    save_unaligned = false
+    sort = true
+
+    BOWTIE2_BUILD ( fasta )
+    BOWTIE2_ALIGN ( input, BOWTIE2_BUILD.out.index, save_unaligned, sort )
+    
+    Channel.fromPath(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+        .map { it -> [[ id:'test', single_end:true ], it] }
+        .join(BOWTIE2_ALIGN.out.bam)
+        .set { input_semibin }
```

Comment:
Isn't it possible to work with the BAM files already available in the test data repository? (less compute time for the tests is better :p)

Reply:
I have tried, but it seems the bam files in the repo can not be used.

Reply:
Hmm strange since those BAMs were also aligned from those FastQ files. What was the error you got?

Reply:
SemiBin needs the contig files (> 5 contigs) and corresponding bam file. But what I found is contig file (1 contig) and the bam file. If there are files that suits the needs in the repo? Thanks!

Reply:
Aaah no I see how you fixed it here! Yes this seems like the ideal solution indeed :)

Reply:
Thanks for clarifying :partying_face: